app/client/src/features/events/Questions/QuestionList/QuestionList.tsx

```diff
@@ -40,14 +40,20 @@ const useStyles = makeStyles((theme) => ({
         flex: '1 1 100%',
     },
     questionActions: {
-        padding: 0,
-        color: theme.palette.primary.light,
+        display: 'flex',
+        justifyContent: 'center',
+        padding: 'theme.spacing(1) 0',
```

Comment:
I don't think this is working properly.  Should be `{theme.spacing(1)}px 0px`.

Reply:
I checked again to fix it and realized that it's not actually needed, so I'll remove it.